src/cclib/parser/daltonparser.py

```diff
@@ -166,7 +174,8 @@ def extract(self, inputfile, line):
             symmetry_atoms = []
             atommasses = []
             for cols in lines:
-                atomnos.append(self.table.number[cols[0]])
+                cols0 = ''.join([i for i in cols[0] if not i.isdigit()]) #remove numbers
```

Comment:
If there are numbers here that need to be remove and crash the parser, could you please add a regression and move this to a separate pull request?